### DISCUSSION TITLE
My plans on making an OS with Cosmo (not higher expectation but at least this?)

### DISCUSSION THREAD

**Discussion by Hasnain12340** [CosmosOS/Cosmos#2486] [Show and tell]
![download (32)](https://user-images.githubusercontent.com/118671904/203731210-eee68917-6908-4e9e-99ba-165efb3b56e1.png)
is atleast this much possible or i can expect higher?

**Comment by terminal-cs** [comment]
you can make it, but it will take time and effort. it isn't a drag and drop experience, this will require: a graphics library (can use cosmos's graphics), a ui system, and image loader

**Reply by Hasnain12340** [reply]
oh ok, so you mean we can make it but harder, yeah?

**Reply by terminal-cs** [reply]
> oh ok, so you mean we can make it but harder, yeah?

yeah, i recomend learning graphics and ui programing first as it will help you a lot instead of relying on other libraries

**Reply by Hasnain12340** [reply]
Thanks for the recommendation

**Comment by AsertCreator** [comment]
you need to implement many things that cosmos doesnt have yet. like, multithreading, image loader, process model, hardware acceleration (i see gaussian blur) and ui library. also, it may be good to split your os into different parts.

**Reply by Hasnain12340** [reply]
Oh thanks
